lib/node/runner/api/resource/resource.go

```diff
@@ -36,8 +36,13 @@ func (l ResourceList) Resource() *hal.Resource {
 		rCollection = append(rCollection, apiResource.Resource())
 	}
 	rl.EmbedCollection("records", rCollection)
-	rl.AddLink("prev", hal.NewLink(l.LinkPrev())) //TODO: set prev/next url
-	rl.AddLink("next", hal.NewLink(l.LinkNext()))
+
+	if l.LinkPrev() != "" {
+		rl.AddLink("prev", hal.NewLink(l.LinkPrev())) //TODO: set prev/next url
```

Comment:
Although there are no more items when this link made,
when the client request thru the link, in the meantime, new items may be created.

What do you think? @anarcher